### ISSUE TITLE
`self.dbms_name` is printed when sqlalchemy engine is connected

### ISSUE THREAD

**Issue by awalker88** [ibmdb/python-ibmdb#625]
Hi, I am using the following code to connect to a DB2 database with `ibm-db==3.0.4` and `ibm-db-sa==0.3.6`. Whenever the last line is run (`conn = engine.connect()`), the string `'DB2/LINUXX8664'` is printed. While not a huge issue, this can clutter up our logs if we're making many connections. 
```
from sqlalchemy import create_engine, exc
env_creds = cloudant_helper.get_db2_credentials(creds, environment)
sqla_url_f = f"db2+ibm_db://{env_creds['username']}:{env_creds['password']}@{env_creds['hostname']}:" \
             f"{env_creds['port']}/{env_creds['database']};{env_creds['ssl']};" \
             f"{env_creds['trustpass']};{env_creds['truststore']};"
try:
    engine = create_engine(sqla_url_f, pool_size=10, max_overflow=20)
    conn = engine.connect()
```
I think I've tracked down the line in question. Inside the class `DB2Dialect`'s function `initialize()`, this line does the printing `print(self.dbms_name)`. On my machine, it is line 707 in `venv/Lib/site-packages/ibm_db_sa/base.py`. Could this print statement be made optional or removed? Let me know if there's a better place to raise the issue. Thanks

**Comment by amukherjee28** [comment]
Hi @awalker88 

The issue you have pointed out is a mistake in the released package of ibm_db_sa where the print line was not removed.

I am aware of the issue and this will be taken care in the next release due next week.

Meanwhile you can install the package from the git repository of ibm_db_sa (https://github.com/ibmdb/python-ibmdbsa) by cloning the code and running the command
python setup.py install

Thanks

**Comment by awalker88** [comment]
Great to hear, thank you!

**Comment by awalker88** [comment]
Hey @amukherjee28, just checking in. Will there be a release in the near future that fixes this issue?

**Comment by amukherjee28** [comment]
Will have the release by 16 and this issue will be fixed.

**Comment by amukherjee28** [comment]
The issue is addressed in the new release 0.3.7 in ibm_db_sa package.
